### PR TITLE
remove cluster_type from hash_stored_account

### PR DESCRIPTION
**Pull request by jeffwashington**
#### Summary of Changes

Fixes #

### CODE REVIEW

File: runtime/src/accounts_db.rs [hunk 1]
```diff
@@ -2878,33 +2878,20 @@ impl AccountsDb {
     pub fn hash_stored_account(
         slot: Slot,
         account: &StoredAccountMeta,
-        cluster_type: &ClusterType,
+        _cluster_type: &ClusterType,
```
Comment (ryoqun) [inline runtime/src/accounts_db.rs:2881]: just making sure: you can stop passing this altogether from the caller (assuming we don't have any immediate plan to change the hashing.).

File: runtime/src/accounts_db.rs [hunk 2]
```diff
@@ -2878,33 +2878,20 @@ impl AccountsDb {
...
-        }
+        Self::blake3_hash_account_data(
+            slot,
+            account.account_meta.lamports,
+            &account.account_meta.owner,
+            account.account_meta.executable,
+            account.account_meta.rent_epoch,
+            account.data,
+            &account.meta.pubkey,
+            include_owner,
```
Comment (ryoqun) [inline runtime/src/accounts_db.rs:2893]: also, you can remove this flag. blake3 and `include_owner` thing even predate the feature activation @mvines brought about. blame @carllin and @ryoqun for cleaning them, respectively. xD

File: runtime/src/accounts_db.rs (hunk 1 above)
Comment (jeffwashington) [reply runtime/src/accounts_db.rs:2881]: yep, phase 2. ;-) roughing out this is what you meant.

**Comment by jeffwashington** [comment]
taken over by @brooksprumo

**Pull request closed without merging**